index.html

```diff
@@ -1,257 +1,264 @@
+            <h3 class="about-me__skills-text">Languages</h3>
+            <img class="about-me__deploy" src="./images/deploy.png" alt="deploy symbol">
+          </div>
+          <div class="about-me__languages">
+            <div class="about-me__skills-tag">
+              <img src="/images/js.png" alt="javascript">
+              <span>JavaScript</span>
+            </div>
+            <div class="about-me__skills-tag">
+              <img class="sixht-sheet__developer-language-img" src="./images/Ellipse-Html.png" alt="html">
+              <span>HTML</span>
+            </div>
+            <div class="about-me__skills-tag">
+              <img class="sixht-sheet__developer-language-img" src="./images/Ellipse-css.png" alt="css">
+              <span>CSS</span>
+            </div>
+          </div>
+          <div class="about-me__skills-language">
+            <h3 class="about-me__skills-text">Frameworks</h3>
+            <img class="about-me__deploy" src="./images/chevron.png" alt="deploy right">
+          </div>
+          <div class="about-me__skills-language">
+            <h3 class="about-me__skills-text">Skills</h3>
+            <img class="about-me__deploy" src="./images/chevron.png" alt="deploy rigth">
+          </div>
+        </div>
+      </div>
     </section>
-    </body>
-  </html>
\ No newline at end of file
+     <footer class="cont-background">
+      <section class="contact-form">
+        <form
+          class="contact-info"
+          action="https://formspree.io/f/mwkyenjw"
+          method="POST"
+        >
+          <h2 class="contact-title">Contact me</h2>
+          <p class="p-contact">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+          <ul class="input-items">
+            <li>
+              <input
+                type="text"
+                name="Name"
+                class="inp-form"
+                maxlength="30"
+                required
+                placeholder="Kalkidan"
+              />
+            </li>
+            <li>
+              <input
+                type="email"
+                name="_replyTo"
+                class="inp-form"
+                id="email"
+                placeholder="kdan2944@gmail.com"
+                required
+              />
+            </li>
+            <li>
+              <textarea
+                name="message"
+                class="text-area"
+                cols="30"
+                rows="10"
+                maxlength="500"
+                placeholder="Write your message here"
+                required
+              ></textarea>
+            </li>
+            <li>
+              <input type="submit" class="btncontact" value="Get It Touch" />
+            </li>
+          </ul>
+          <img class="end-line" src="./images/Shape.png" alt="" />
+        </form>
```

Comment:
- [ ] You are almost there in your Form section styling. Kindly make sure you stick as much as possible to your chosen Figma design padding as margins.
- [ ] Also, kindly ensure that you position your form submit button in the middle as it is in the Figma design this will make your project look more professional and consistent.

**This is how your design currently look like:**

![image](https://user-images.githubusercontent.com/86318284/179285521-fe7b509d-bb20-4481-98c8-ee2b9521162b.png)

**This is the desired Figma design:**

![image](https://user-images.githubusercontent.com/86318284/179285636-42f5d9ea-f66a-440d-8426-ad57fbcbd026.png)